core/src/main/resources/migration/tenant/4.0-2019-04-10-OC-10430.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+
+
+	<changeSet author="jkeremian" id="2019-04-10-OC-10430-1">
+		<sql>
+			UPDATE audit_log_event
+			SET user_id='1' WHERE user_id ='0' or user_id is Null;
+		</sql>
+	</changeSet>
```

Comment:
can you move this to a separate file so it can be easily removed for production?